WineCellar/WineCellar.Controller/Data.cs

```diff
@@ -24,38 +30,53 @@ public static List<IWineData> GetWineData()
+                new WineData {ID = 7, Name = "Domaine Andau", HarvestYear = 2019, Rating = 7, Type = "Grüner Veltliner", Alcohol = 12, OriginCountry = "Oostenrijk", Stock = 3, Taste = new string[] { "Elegant en Iets Kruidig"}, StorageLocation = new []{"A3.6","A18.6","B3.20","B3.21","B3.22","B3.23","B3.24","C2.2"}, BuyPrice = 7.99, SellPrice = 23.99, Description = "In de geur van deze super frisse Grüner Veltiner ruik je pitten van perziken en een beetje een kruidige geur van witte peper. De smaak is elegant smaak met mooie, verfijnde zuren."},
+                new WineData {ID = 8, Name = "La Palma", HarvestYear = 2019, Rating = 4, Type = "Chardonnay", Alcohol = 2, OriginCountry = "Chili", Stock = 2, Taste = new string[] { "Soepel, Zacht en Tropisch Fruit"}, StorageLocation = new []{"A3.6","A18.6","B3.20","B3.21","B3.22","B3.23","B3.24","C2.2"}, BuyPrice = 5.99, SellPrice = 23.99, Description = "Carmenere is de nationale druif van Chili. Oorspronkelijk komt hij uit Bordeaux, maar daar staat hij bijna niet meer aangeplant. Sterker nog, men dacht dat hij was uitgestorven. Totdat de Chilenen hem herontdekten en er achter kwamen dat de omstandigheden in Chili ideaal zijn voor de aanplant van deze druif. De wijngaarden van La Palma profiteren van grote verschillen tussen dag en nacht temperatuur wat zorgt voor een hoge kwaliteit. In deze carmenere proef je rode bessen, pruimen, bramen en cassis. Een toegankelijke wijn, geschikt voor ieder moment."},
+                new WineData {ID = 9, Name = "La Palma", HarvestYear = 2020, Rating = 2, Type = "Carmenère", Alcohol = 80, OriginCountry = "Chili", Stock = 53, Taste = new string[] { "Bes, Pruim en Braam"}, StorageLocation = new []{"A3.6","A18.6","B3.20","B3.21","B3.22","B3.23","B3.24","C2.2"}, BuyPrice = 5.99, SellPrice = 23.99, Description = "De La Palma wijnen worden gemaakt door het Chileense wijnbedrijf Viña la Rosa. Alle wijnen van Viña la Rosa worden duurzaam geproduceerd met respect voor werknemers, leveranciers en nabijgelegen gemeenschap. In deze 100% chardonnay proef je de smaak van tropisch fruit. Denk aan ananas, banaan en een vleugje limoen. Daarnaast ontdek je perzik, gele appel en geroosterde hazelnoten. Hij is rijk, vol en romig. Juist daarom lekker bij gegrilde kip, gerookte zalm of varkenshaas met champignonroomsaus."}
             };
         }
-        
+
         //Get all the wine data from the database
         public static async Task<List<IWineData>> GetAllWines()
         {
             var wineRepo = await DataAccess.WineRepo.GetAll();
+
             //convert the result to a list of wine data
             var wineData = new List<IWineData>();
-            
+
+            string[] wineNotes = Array.Empty<string>();
+            string[] storageLocations = Array.Empty<string>();
```

Comment:
Kan dit niet in de loop gezet worden? Hierdoor hoeft "storageLocations = Array.Empty<string>();" niet meer?